src/pages/CustomLessonsDisplay/CustomLessonsDisplay.js

```diff
@@ -52,8 +49,15 @@ const CustomLessonsDisplay = props => {
       setDisplayTemplate(lesson.lessonTemplate);
     }
 
+    let deploymentAccountIds = new Set();
+    for (const dueDate in lesson.dueDates) {
+      for (const deploymentAccount of lesson.dueDates[dueDate]) {
+        deploymentAccountIds.add(deploymentAccount);
+      }
+    }
```

Comment:
same as above for using `.forEach` instead of `for`

Reply:
love you dad